web/src/components/Callout.tsx

```diff
@@ -89,11 +91,19 @@ export const Callout = (props: PropsWithChildren<Props>): ReactNode => {
               {headingText}
             </span>
           </div>
-
-          <div className="margin-top-105 text-primary-darker">{props.children}</div>
+          {props.children && <div className="margin-top-105 text-primary-darker">{props.children}</div>}
         </>
       ) : (
-        <div className="text-primary-darker">{props.children}</div>
+        <div className="text-primary-darker flex">
+          {showIconInBody && (
```

Comment:
Just to make sure I understand, we only want to show the icon in body if we are not showing the header?

Reply:
This is the only case where we have an icon in the body. It's a new pattern, so that's why I made it a new prop.